pkg/harvester/edit/kubevirt.io.virtualmachine/VirtualMachineSSHKey.vue

```diff
@@ -134,6 +141,42 @@ export default {
       this.$modal.hide(this.randomStr);
     },
 
+    async createNamespaceIfNeeded() {
+      if (!this.createNamespace || this.disableCreate) {
+        return;
+      }
+
+      try {
+        const namespaces = await this.$store.dispatch('harvester/findAll', { type: NAMESPACE });
+
+        const exists = namespaces?.find(n => n.name === this.namespace);
+
+        if (!exists) {
+          const ns = await this.$store.dispatch('harvester/createNamespace', { name: this.namespace }, { root: true });
+
+          ns.applyDefaults();
+          await ns.save();
+        }
+      } catch {}
```

Comment:
Should we handle this exception to log or something?

Reply:
God catch thanks; this is a bug, I'm removing the try-catch from `createNamespaceIfNeeded` since the errors should be catch in the `save` function;